docs/developer-guide/syntax-reference.rst

```diff
@@ -1153,6 +1153,17 @@ Example:
       WHERE ROWTIME >= 1510923225000
         AND ROWTIME <= 1510923228000;
 
+When writing logical expressions using `ROWTIME`, ISO-8061 formatted datestrings can also be used to represent dates.
+For example, the above query is equivalent to the following:
+
+.. code:: sql
+    SELECT * FROM pageviews
+          WHERE ROWTIME >= '2017-11-17 04:53:45'
+            AND ROWTIME <= '2017-11-17 04:53:48';
+
+If the datestring is inexact, the rest of the timestamp is assumed to be 0.
+For example, `ROWTIME = `2019-07-30 11:00` is equivalent to `ROWTIME = `2019-07-30 11:00:00`.
```

Comment:
```suggestion
For example, ``ROWTIME = 2019-07-30 11:00`` is equivalent to ``ROWTIME = 2019-07-30 11:00:00``.
```

Reply:
i might explicitly call out here that all unspecified elements of the full pattern are right-padded with zeros

Reply:
somewhere in here i would expect to see a mention of timezone handling too, as this always trips people up.... i.e. in what TZ are your iso-8601 strings interpreted ? that of teh running jvm or utc or ....?